src/app.rs

```diff
@@ -315,7 +315,7 @@ impl<'a, 'b, T> ApplicationBuilder<'a, 'b, T> {
                     message
                 ))
             })
-            .level(LevelFilter::Debug)
+            .level(LevelFilter::Warn)
```

Comment:
We should make this controllable from the commandline (you can ignore it in this PR though).